be/src/olap/storage_engine.h

```diff
@@ -363,6 +378,8 @@ class StorageEngine {
     // used to control the the process of converting old data
     RowsetTypePB _compaction_rowset_type;
 
+    HeartbeatFlags* _heartbeat_flags;
```

Comment:
```suggestion
    HeartbeatFlags* _heartbeat_flags = nullptr;
```

Reply:
I will init it to nullptr in constructor.